apisix/cli/ngx_tpl.lua

```diff
@@ -51,6 +51,9 @@ worker_shutdown_timeout {* worker_shutdown_timeout *};
 env APISIX_PROFILE;
 env PATH; # for searching external plugin runner's binary
 
+# reserved environment variables for configuration
+env APISIX_DEPLOYMENT_ETCD_HOST;
```

Comment:
Can we find a way to cover it in the CI? So we don't need to spend time verifying it locally next time.